api/hosts/host.gen.go

```diff
@@ -24,6 +24,7 @@ type Host struct {
 	Type              string                 `json:"type,omitempty"`
 	HostSetIds        []string               `json:"host_set_ids,omitempty"`
 	Attributes        map[string]interface{} `json:"attributes,omitempty"`
+	Address           string                 `json:"address,omitempty"`
```

Comment:
This might be a little unfortunate.  I wonder if there is a way to comment on this generated field to notify SDK users that this is a read only field and that they should use the Option to set the address field or do we just hope the user tests or figures this out...

Reply:
Now that we've resolved to change this to preferred_address do you think this resolves the issue?

Reply:
Have we decided on renaming it as `prefered_address`?

I wonder if we should have a new read only resource that is called something like "endpoint".   This resource would have the `host_id` that it came from, the `host_set` which is used to determine the address from that `host` and the address selected from that collection of addresses associated with that host.  The reason I think this might make sense is that it feels strange to have a single resource return different values depending on if we have `GET v1/hosts?host_set=foo` and `GET v1/hosts?host_set=bar`.

@jefferai, opinion?